eth_test_parser/src/trie_builder.rs

```diff
@@ -169,23 +167,13 @@ impl From<TestBody> for GenerationInputs {
     }
 }
 
-fn get_pre_account_hashes(
-    account_address: &H160,
-    account: &PreAccount,
-    storage_tries: &[(H160, PartialTrie)],
-) -> (H256, H256) {
-    match account.code.0.is_empty() {
-        false => (
-            hash(&account.code.0),
-            storage_tries
-                .iter()
-                .find(|(addr, _)| account_address == addr)
-                .unwrap()
-                .1
-                .calc_hash(),
-        ),
-        true => (H256::zero(), H256::zero()),
```

Comment:
For an EOA, I think we want the hash of the empty code string and the hash of the empty trie. Seems simplest to not treat it as a special case.